Berksfile

```diff
@@ -5,3 +5,5 @@ metadata
 group :test do
   cookbook 'octopus-deploy-test', path: './test/cookbooks/octopus-deploy-test'
 end
+
+cookbook 'windows_firewall'
```

Comment:
This shouldnt be needed, it will pick it up from the metadata file.  Only test dependencies need to be pushed here